### PR TITLE
Added SNMP TRAP Auth Handler for Edgecos edge-core switches

### PR DESCRIPTION
**Pull request by jozefrebjak**
DO NOT DELETE THIS TEXT

#### Please note

> Please read this information carefully. You can run `./scripts/pre-commit.php` to check your code before submitting.

- [ ] Have you followed our [code guidelines?](http://docs.librenms.org/Developing/Code-Guidelines/)

#### Testers

If you would like to test this pull request then please run: `./scripts/github-apply <pr_id>`, i.e `./scripts/github-apply 5926`
After you are done testing, you can remove the changes with `./scripts/github-remove`.  If there are schema changes, you can ask on discord how to revert.

Supported models: ES3528,ECS4210

### CODE REVIEW

**Comment by murrant** [comment]
@jozefrebjak I can't believe you still are using the webui to upload changes.  If you need any help with git ask in #devel

File: tests/Feature/SnmpTraps/EdgeCosFailedUserLoginTest.php [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+
+use App\Models\Device;
+use App\Models\Ipv4Address;
+use LibreNMS\Snmptrap\Dispatcher;
+use LibreNMS\Snmptrap\Trap;
+use LibreNMS\Tests\LaravelTestCase;
+
+class EdgeCosFailedUserLoginTest extends LaravelTestCase
+{
+    public function test3528Series()
```
Comment (murrant) [inline tests/Feature/SnmpTraps/EdgeCosFailedUserLoginTest.php]: You create a device in the DB, but you did not set `use DatabaseTransactions;`

**Comment by laf** [comment]
@jozefrebjak Any update on the tests?

**Comment by laf** [comment]
Closing for now due to no activity.

**Pull request closed without merging**